app/models/value_definition.rb

```diff
@@ -4,10 +4,20 @@
 class ValueDefinition < ApplicationRecord
   belongs_to :benchmark, class_name: 'Xccdf::Benchmark'
 
-  POSSIBLE_VALUE_TYPES = %w[string number bool].freeze
+  POSSIBLE_VALUE_TYPES = %w[string number boolean].freeze
```

Comment:
Why? How does it look like in the XCCDF specs? cc @romanblanco

Reply:
It's `boolean` in the XCCDF specs. I just noticed it because during the values import, boolean types weren't being imported correctly since it wasn't a possible value type.

Reply:
@marleystipich2 is right, this was a mistake

![image](https://user-images.githubusercontent.com/1187051/207530664-8e21e78d-516d-4925-bf1d-2b2a39a3c423.png)